exporter/clickhousetracesexporter/migrations/000012_init_signoz_error_index_v2.up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS signoz_traces.signoz_error_index_v2 (
+  timestamp DateTime64(9) CODEC(DoubleDelta, LZ4),
+  errorID FixedString(32) CODEC(ZSTD(1)),
+  groupID FixedString(32) CODEC(ZSTD(1)),
+  traceID FixedString(32) CODEC(ZSTD(1)),
+  spanID String CODEC(ZSTD(1)),
+  serviceName LowCardinality(String) CODEC(ZSTD(1)),
+  exceptionType LowCardinality(String) CODEC(ZSTD(1)),
+  exceptionMessage String CODEC(ZSTD(1)),
+  exceptionStacktrace String CODEC(ZSTD(1)),
+  exceptionEscaped bool CODEC(T64, ZSTD(1)),
+  INDEX idx_error_id errorID TYPE bloom_filter GRANULARITY 4
+) ENGINE MergeTree()
+PARTITION BY toDate(timestamp)
+ORDER BY (timestamp, groupID)
```

Comment:
@makeavish this query will be costly if we try to filter by `groupID` without passing `timestamp`. Do we have any such API that does like that?

Reply:
No we don't have such a query right now.